.github/workflows/e2e_translation_classroom_and_core_features.yml

```diff
@@ -46,20 +46,15 @@ jobs:
       - name: Build Webpack
         if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
         run: python -m scripts.build --prod_env
-      - name: Run Exploration Translation Tab E2E Test
+      - name: Run Diagnostic Test Player Page E2E Test
         if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
-        run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-build --skip-install --suite="explorationTranslationTab" --prod_env
```

Comment:
These lines are removed to run the workflow faster in the CI. I will add them back after fixing the test.